service/gradle/task-dependencies.gradle

```diff
@@ -18,3 +18,6 @@ for (task in TASKS_ONLY_IF_COMPILE_JAVA) {
 
 // Explicit dep to avoid accidental dep deprecation warning.
 tasks.processResources.dependsOn(generateVersionProperties)
+
+// Status and version endpoint tests added to regression test
+tasks.regressionTests.dependsOn(generateVersionProperties)
```

Comment:
Regression tests are a way to check that cohort queries do not change over time for a given dataset ([docs](https://github.com/DataBiosphere/tanagra/blob/main/docs/REGRESSION_TESTING.md)). They should not include any specific endpoint tests.

Reply:
Was trying to avoid adding another test tag, probably not a good idea. changing